Guide: How to Create Your First Mock Server

Aman333Saxena opened the pull request:
A blog for the "How-To Guides" category, about How to Create Your First Mock Server simulating APIs. It consists of all the extensive details, step-by-step instructions with images and resources links.

Update1: Author Bio and the Headshot Image

hectorkambow commented:
@Aman333Saxena if you could please make sure you have updated your repo to the most recent - and/or follow the instructions to upload your article and image file - they do not seem to be in the correct place.
images don't need to be in a separate folder

Aman333Saxena commented:
Hello, @hectorkambow 
I have added all the images and the article in the same folder as you have asked.  The current Repo also looks clean.  
I have updated image references as well. All good to go :)

hectorkambow commented:
Good afternoon @manibatra  - could you please give this article a tech review when you get a chance. Thank you 🚀

manibatra commented on articles/guide-to-create-mock-server/index.md:
```diff
@@ -0,0 +1,90 @@
+![Usage Image](/engineering-education/guide-to-create-mock-server/image01.png) *Fig 2: www.mock-server.com*
+
+- Mock Servers easily recreate all types of responses for HTTP dependencies such as REST or RPC services to test applications easily and effectively. They isolate the system-under-test to ensure tests run reliably and only fail when there is a genuine bug.
+- These servers easily setup mock responses independently for each test to ensure test data is encapsulated with each test. Avoid sharing data between tests that are difficult to manage and maintain this also decreases the risks of tests infecting each other.
```
`They avoid sharing data between tests that are difficult to manage and maintain. This also decreases the risks of tests infecting each other.`

manibatra commented on articles/guide-to-create-mock-server/index.md:
```diff
@@ -0,0 +1,90 @@
+![Set up Image](/engineering-education/guide-to-create-mock-server/image4.png)
+
+- In the **Set up the mock server** tab, you can configure your mock server.
+1. Enter the name of the mock.
```
name of the mock server?

manibatra commented on articles/guide-to-create-mock-server/index.md:
```diff
@@ -0,0 +1,90 @@
+
+- If the setting process went well without an error, you'll be able to see your Mock Server in the left corner on the **Collections** panel.
+
+You can view and search the details of calls to your mock servers using the mock call log. Open a mock from the Postman app by clicking it in Collections, in APIs, or by switching to Browse > Mocks and clicking the mock name.
```
Open a mock server?

manibatra commented on articles/guide-to-create-mock-server/index.md:
```diff
@@ -0,0 +1,90 @@
+
+![Request Image](/engineering-education/guide-to-create-mock-server/image6.png)
+
+- In the left-hand corner, you will see **Setting Option(3 Dots)** in the Collections panel. On clicking that, you'll be able to see several options. Choose **Add Request**, by which a large number of Request Methods, (i.e. GET, POST) will be made available to you, choose whatever you need. Make sure you have an example saved for the request in the collection you have the mock connected to.
```
see the **Setting**

Aman333Saxena commented:
@hectorkambow @manibatra 
I have fixed all the requested changes. It looks great after the edits. 
Thank you so much for an in-detail review. 🚀 ✨
